documents-storage-jcr/src/main/java/org/exoplatform/documents/storage/jcr/JCRDocumentFileStorage.java

```diff
@@ -699,10 +729,7 @@ public AbstractNode duplicateDocument(long ownerId, String fileId, String prefix
         newNode = duplicateItem(oldNode, parentNode, parentNode, prefixClone);
         parentNode.save();
       }
-      AutoVersionService autoVersionService = CommonsUtils.getService(AutoVersionService.class);
-      if (autoVersionService != null) {
-        autoVersionService.autoVersion(newNode);
-      }
+      autoVersion(newNode);
```

Comment:
COuld you use org.exoplatform.services.jcr.ext.utils.VersionHistoryUtils#createVersion to create the new version ?